mscore/pianolevelsfilter.cpp

```diff
@@ -66,6 +68,59 @@ void PianoLevelFilterLen::setValue(Staff* staff, Note* note, NoteEvent* evt, int
       score->endCmd();
       }
 
+//---------------------------------------------------------
+//   maxRange
+//---------------------------------------------------------
+
+int PianoLevelFilterLenOfftime::maxRange()
+      {
+            return MScore::division;
```

Comment:
Too much indent here, same for the next function. Actually these two functions can also be defined directly in .h file.

Reply:
These functions depend on MScore which is defined in 
#include "libmscore/score.h"
If I move them to the header file, I'd need to move the include there too.